highfidelity/hifi
# Case 20723, Case 20334: Internally, replace all overlays with local entities

https://highfidelity.manuscript.com/f/cases/20723/
https://highfidelity.manuscript.com/f/cases/20334/HMD-Login-Zone-tilted-keyboard

Test plan:
- verify that the following work as in master:
    - login screen (desktop and HMD, I think?)
    - tablet (as much as possible.  include: create, home button highlighting, general clicking etc., resizing your avatar)
    - avatar loading spheres (they should blink and change color and disappear)
    - context overlay (floating "i" when you right click on a marketplace item)
    - keyboard focus highlight (when you click on a web entity or overlay, it should have a yellow outline, except for the tablet)
    - create gizmos (in this PR, they will be unlit.  the fact that they were lit was a bug) (should highlight on mouse over, should be clickable and work)
    - 3D keyboard + styluses (all related test rail cases and the test plan from case 20334)
    - hand lasers (rendering, interaction)
    - pal spheres
    - important scripts that make overlays (the shapes app from the marketplace, for example)
- run all tests in Nitpick from this PR: https://github.com/highfidelity/hifi_tests/pull/335
